code/modules/xenoarcheaology/tools/ano_device_battery.dm

```diff
@@ -156,6 +156,9 @@
 	else if(href_list["shutdown"])
 		shutdown_emission()
 		. = TOPIC_REFRESH
+	else if(href_list["refresh"])
+		interact(user)
```

Comment:
Returning `TOPIC_REFRESH` should be instead of calling `interact`. Calling `interact` does nothing on this machine.